diff --git a/host/arch/x86/lib/crossystem_arch.c b/host/arch/x86/lib/crossystem_arch.c
index f3e631a..31e34f0 100644
--- a/host/arch/x86/lib/crossystem_arch.c
+++ b/host/arch/x86/lib/crossystem_arch.c
@@ -70,7 +70,11 @@
 #define GPIO_SIGNAL_TYPE_PHASE_ENFORCEMENT 4
 
 /* Base name for ACPI files */
-#define ACPI_BASE_PATH "/sys/devices/platform/chromeos_acpi"
+//#define ACPI_BASE_PATH "/sys/devices/platform/chromeos_acpi"
+
+/* Kernel registers device at */
+#define ACPI_BASE_PATH "/sys/devices/platform/GOOG0016:00"
+
 /* Paths for frequently used ACPI files */
 #define ACPI_BINF_PATH ACPI_BASE_PATH "/BINF"
 #define ACPI_CHNV_PATH ACPI_BASE_PATH "/CHNV"
